obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+- agasc_id:
+  - 978851616
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978853160
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978986608
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 1043726440
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 1043726952
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 639635448
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:023:22:13:56.794
+  status: 1
+- agasc_id:
+  - 639635448
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:024:04:12:45.955
+  status: 1
+- agasc_id:
+  - 395055600
+  comments: UNKNOWN: Never acquired. Unknown issue with observation. Presume star is OK.
+  mp_starcat_time: 2009:040:05:37:54.381
+  status: 1
+- agasc_id:
+  - 720504096
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:049:20:22:15.791
+  status: 1
+- agasc_id:
+  - 185871616
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:073:15:34:21.422
+  status: 1
+- agasc_id:
+  - 616699752
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:290:21:44:31.992
+  status: 1
+- agasc_id:
+  - 373564272
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:335:11:29:27.547
+  status: 1
+- agasc_id:
+  - 161357600
+  comments: Almost never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:345:18:09:20.636
+  status: 1
+- agasc_id:
+  - 1130649544
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:355:12:36:00.392
+  status: 1
+- agasc_id:
+  - 1130649544
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:356:09:31:17.829
+  status: 1
+- agasc_id:
+  - 1130649544
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:357:03:32:22.942
+  status: 1
+- agasc_id:
+  - 1130649544
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:357:23:28:34.822
+  status: 1
+- agasc_id:
+  - 497824592
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2010:004:19:53:30.715
+  status: 1
+- agasc_id:
+  - 391120072
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2010:042:06:34:25.316
+  status: 1
+- agasc_id:
+  - 190716896
+  comments: Faint star. Tracking affected by imposter. Marginally acceptable
+  mp_starcat_time: 2010:084:04:33:25.441
+  status: 0
+- agasc_id:
+  - 805966472
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2010:089:12:06:07.502
+  status: 1
+- agasc_id:
+  - 962998672
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2010:142:09:25:14.003
+  status: 1
+- agasc_id:
+  - 904010616
+  comments: Faint star. Near MAXMAG. Marginally acceptable
+  mp_starcat_time: 2010:221:16:22:00.639
+  status: 0
```

Comment:
I'm a bit confused here.  If this is status=0 why bother with the manual addition in 'mags'?

Reply:
Good catch. If the status is 0, it should not be in 'bad' or 'mags'